opta/commands/secret.py

```diff
@@ -102,7 +103,7 @@ def view(
             f" already there - update it via secret update."
         )
 
-    print(secrets[secret])
+    logger.info(secrets[secret])
```

Comment:
I don't think we should use the logger to print the secrets since that would make the line contain extra info (maybe?). I think it would also send the secret data to datadog, which is definitely not something we want.